stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/usecase/SendMessageWithFilesTest.kt

```diff
@@ -145,40 +145,6 @@ internal class SendMessageWithFilesTest : BaseDomainTest2() {
         }
     }
 
-    @Test
-    fun `Upload attachment with error should be configurable`() = testCoroutines.scope.runBlockingTest {
```

Comment:
They should not be configurable, because `result.data()` should not be available. In the `sendMessage` the `Attachment` is still available and we can configure it there.